src/scratch/BlockMenus.as

```diff
@@ -533,9 +533,28 @@ public class BlockMenus implements DragClient {
 		var m:Menu = new Menu(null, 'proc');
 		addGenericBlockItems(m);
 		m.addItem('edit', editProcSpec);
+		if (block.op==Specs.CALL) {
+			/// TLF: alternative text? -maybe "show definition"...?
+			m.addItem('go to definition', gotoProcSpec);
```

Comment:
Yup, I'll do that in a bit - that's the wording I'd come to prefer myself. :)
I did wonder about whether to merge this with #861 (the highlight PR) in some way - maybe, along with the "highlight callers" item (don't really like 'callers'...), having a "highlight define script" item, which actually also jumps to the script like this.
But I guess that PR is a considerably larger change, and this one does stand better by itself, so...